herbert_core/utilities/classes/@hashable/private/to_hashable_array_.m

```diff
@@ -32,12 +36,22 @@
         if isa(val,'hashable')
             [val,hash,new_hash] = build_hash(val);
             if new_hash
+                obj(j).do_check_combo_arg = false; % if this operation is
+                % part of constructor we do not need to check combo_values.
+                % It may fail.
                 obj(j).(field_name) = val;
+                obj(j).do_check_combo_arg = true;
             end
+            %-----> hash class dependent block
+            % the operation below work only for MD5 hash in string form
+            % if we ever decided to switch to uint64-bit hash, operations
+            % with container modification become an order of magnitude
+            % faster. This block must change is such case.
             if iscell(hash) % use array of hashes as source for final hash
                 hash = strjoin(hash ,'');
             end
-            tm = typecast(hash,'uint8');
+            tm = uint8(hash); % use typecast for numeric hash.
```

Comment:
This comment does not make sense. You aren't using `typecast`

Reply:
well, if you read sentences 45-49, you will see the context. I have clarified it here too.